tools/icqsol_rotate_shape/icqsol_rotate_shape.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+import argparse
+import shutil
+
+from icqsol.shapes.icqShapeManager import ShapeManager
+from icqsol import util
+import icqsol_utils
+import numpy
```

Comment:
Do not import packages like numpy into the Galaxy wrappers.  The complexities of using bumpy and other similar packages must be encapsulated within classes in the icqsol package - see below.